src/posts/lucky_1.0.0-rc1.cr

```diff
@@ -0,0 +1,239 @@
+    This library helped to make pages feel even faster, and who doesn't love speed?
+    Lucky included this library by default, but it has since been deprecated in favor
+    of [Turbo](https://turbo.hotwired.dev/). Although Lucky doesn't include Turbo currently,
+    we would still like to encourage users to give it a shot, and let us know if this
+    is something you'd like to see included by default.
+
+    Along those same lines, Lucky also came out-of-the-box with [Laravel-Mix](https://laravel-mix.com/)
+    which is "An elegant wrapper around Webpack". This made bringing javascript to Lucky
+    seemless, and easy. However, the Laravel framework has [moved to Vite](https://laravel-news.com/vite-is-the-default-frontend-asset-bundler-for-laravel-applications)
+    as their default front-end. Laravel-Mix may or may not stick around, but with the
+    uncertianty, the 1.0.0-rc1 release opens up the possibility to use [Vite](https://vitejs.dev/)
+    while keeping with the type-safe traditions Lucky brings.
+
+    So why even include a javascript build system with Lucky?
+
+    One of Lucky's goals is to be type-safe and help catch bugs in development. This
+    is done many ways, but one of the most important ways is when it comes to loading
+    your assets in production. By utilizing a generated manifest file, Crystal can
+    generate an internal Hash at compile-time that can be referenced in the app.
+
+    This means in development, you can do `img src: asset("images/logo.svg")`, and
+    Lucky knows this file is located at `public/assets/images/logo.svg`. In production,
+    you may need to use a cache buster, or even a CDN where the filename may change to
+    say `assets.mysite.com/images/logo-abc123xyz.svg`. If this file is missing, then
+    your app will not compile. That means that last second production deploy you're
+    doing on a Friday before a 3-day weekend won't have you back in the "office"
+    doing another deploy because the file isn't showing.
+
+    ### Email layouts and callbacks
+
+    Carbon 0.3.0 has been released with a few neat updates. The first being
+    the ability to set email layouts. Just like your standard views, you may
+    have a very similar wrapper on your emails like contact footer info, or
+    your logo.
+
+    To specify your layout, you'll add the `layout` macro, and pass the name
+    of the folder where your layout template file is located.
+
+    ```crystal
+    # src/emails/welcome_email.cr
+    class WelcomeEmail < BaseEmail
+      def initialize(@recipient : Carbon::Emailable)
+      end
+
+      to @recipient
+      from "info@myapp.com"
+      subject "Welcome to the site!"
+      layout my_email_layout
+      templates html, text
+    end
+    ```
+
+    Then in your templates, you can add `my_email_layout/layout.ecr`,
+    and use the `content` method to yield your email body
+
+    ```
+    # src/emails/templates/my_email_layout/layout.ecr
```

Comment:
I'm not sure, but should this have the content type in the filename?

```suggestion
    # src/emails/templates/my_email_layout/layout.html.ecr
```

Reply:
actually it doesn't :joy: it should though lol